go.mod

```
@@ -1,18 +1,18 @@
 module github.com/kubeflow/mpi-operator
 
-go 1.22.0
+go 1.23.0
```

```suggestion
go 1.23
```

Could we eliminate the patch version?

---

done